### PR TITLE
Create moves.js

### PR DESCRIPTION
**Pull request by Ludicrousity**
Add this in a folder called Eternal Pokemon please Zard. Pet Mod being made.

### CODE REVIEW

**Comment by Ludicrousity** [comment]
Dunno what happened to the indents... they were fine when I made it :/

File: mods/moves.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	             	priority: 0,
+	             	flags: {protect: 1, mirror: 1},
+	            	critRatio: 0,
+	             	secondary: false,
+	            	target: "normal",
+	            	type: "Electric",
+                zMovePower: 180
+	      },
+// insert Eternal Eevee's move here //
+                       zMovePower: 180
```
Comment (Spandamn) [inline mods/moves.js]: This will crash

File: mods/moves.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
+'use strict';
```
Comment (Spandamn) [inline mods/moves.js]: Run this through jsbeautifier

Comment (Ludicrousity) [inline mods/moves.js]: have done. doesn't look right

Comment (Ludicrousity) [inline mods/moves.js]: Doesn't look right now

**Comment by charizard8888** [comment]
We already have a folder for this xd

**Pull request merged**